### PR TITLE
add start of container to README

### PR DESCRIPTION
**Pull request by georgebatalinski**
Would it be beneficial to have this information right on README?

### CODE REVIEW

**Comment by nerdcha** [comment]
Thanks, that's a great idea!

File: README.md [hunk 1]
```diff
@@ -6,4 +6,13 @@ This is the Dockerfile (etc.) used for building the image that runs python scrip
 
 Because the build process of this image has become quite lengthy, we've split it into three. The base images are at [here](https://github.com/Kaggle/docker-python0) and [here](https://github.com/Kaggle/docker-python1).
 
+**To start container [Learn more](http://blog.kaggle.com/2016/02/05/how-to-get-started-with-data-science-in-containers/comment-page-1/)
```
Comment (nerdcha) [inline README.md]: I'd prefer to drop the double asterisk at the start, and have something like `For a guide to using the container, see [our blog](http://blog.kaggle.com/2016/02/05/how-to-get-started-with-data-science-in-containers).`

File: README.md [hunk 2]
```diff
@@ -6,4 +6,13 @@ This is the Dockerfile (etc.) used for building the image that runs python scrip
...
+
+Mac
```
Comment (nerdcha) [inline README.md]: I don't think this part is so helpful—if we start explaining different host/use-case combinations in this readme, we'd run into a combinatorial explosion pretty quickly.

Comment (georgebatalinski) [inline README.md]: Are you referring to the -p 8888:8888? and --ip="0.0.0.0"
Reason: The default jupyter notebook is setup to start on that port, so unless we have a mapping to local port -- user won't be able to access the notebook.

thoughts?

Comment (nerdcha) [inline README.md]: No, more that the linked blogpost explains this stuff in enough detail, and I don't feel we need to reproduce the advice here…?

**Pull request closed without merging**